CorsixTH/Lua/persistance.lua

```diff
@@ -260,10 +260,13 @@ function SaveGameFile(filename)
   f:close()
 end
 
+--! Loads the game's save into the game
```

Comment:
Nice recursion putting games in games :) 

As you're loading here, perhaps avoid mention 'save'?  "loaded file' is clear enough I think.
Also, it misses a description for `data` (which is somewhat too generic to my taste, literally everything is 'data').